back/spec/acceptance/projects_spec.rb

```diff
@@ -1505,7 +1505,7 @@
       json_response = json_parse(response_body)
 
       current_phase_ids = json_response[:data].filter_map { |d| d.dig(:relationships, :current_phase, :data, :id) }
-      included_phase_ids = json_response[:included].select { |d| d[:type] == 'phase' }.pluck(:id)
+      included_phase_ids = json_response[:included].select { |d| d[:type] == 'phase_mini' }.pluck(:id)
```

Comment:
Maybe we should split this file up, by creating a `projects_mini_spec` or something like that? It's getting really really long

Reply:
That's not a bad idea 👍

Reply:
https://www.notion.so/govocal/Break-new-projects-acceptance-tests-out-into-separate-file-14a9663b7b2680d28877e83d63c54d3c?pvs=4